maintainers/maintainer-list.nix

```diff
@@ -7921,6 +7921,12 @@
     githubId = 720864;
     name = "Sébastien Bourdeauducq";
   };
+  sbond75 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add sbond75``.